builder/dockerignore/dockerignore_test.go

```diff
@@ -1,69 +0,0 @@
-package dockerignore // import "github.com/docker/docker/builder/dockerignore"
```

Comment:
Can you combine the "use buildkit dockerignore" with the vendor commit?

Also should add a "deprecated.go" that deprecated it, but aliases it to to the new location (to help the transition); there's quite some consumers of it; https://grep.app/search?current=2&q=github.com/docker/docker/builder/dockerignore

Reply:
This is not `pkg/` but fine

Reply:
Does it make a difference? afaik the `pkg/` prefix was a _convention_, but now discouraged and deprecated